packages/unchained-client/src/evm/parser/arbitrumBridge.ts

```diff
@@ -140,6 +140,17 @@ export class Parser implements SubParser<Tx> {
               },
             })
           }
+          case this.l2ArbitrumGatewayAbi.getFunction('finalizeInboundTransfer')!.selector:
+            return await Promise.resolve({
+              data: {
+                ...data,
+                // A finalized inbound transfer on the Arb side is an Eth -> Arb deposit, on the Eth side it's an Arb -> Eth withdrawal
```

Comment:
Note, that's not *entirely* true:
https://docs.arbitrum.io/build-decentralized-apps/token-bridging/token-bridge-erc20

> Withdrawals

> On Arbitrum, a user calls L2GatewayRouter.outBoundTransfer, which in turn calls outBoundTransfer on arbSomeERC20Token's gateway (i.e., L2ERC20Gateway).
> This burns arbSomeERC20Token tokens, and calls ArbSys with an encoded message to L1ERC20Gateway.finalizeInboundTransfer, which will be eventually executed on L1.
> After the dispute window expires and the assertion with the user's transaction is confirmed, a user can call Outbox.executeTransaction, which in turn calls the encoded L1ERC20Gateway.finalizeInboundTransfer message, releasing the user's tokens from the L1ERC20Gateway contract's escrow.

In other words, for deposits, `finalizeInboundTransfer` *is* the outbound Tx. However, for withdraws, since they require a claim, `finalizeInboundTransfer` is only internal. Only kept it for the sake of following the same pattern as above:

https://github.com/shapeshift/web/blob/5e285df9b921118408ad3484622441fc18761a79/packages/unchained-client/src/evm/parser/arbitrumBridge.ts#L100-L103